src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformerTest.php

```diff
@@ -72,4 +73,15 @@ public function testReverseTransformEmpty()
 
         $this->assertNull($transformer->reverseTransform(''));
     }
+
+    public function testFloatToIntConversionMismatchOnReversTransform()
+    {
+        $transformer = new MoneyToLocalizedStringTransformer(null, null, null, 100);
+
+        if (!function_exists('bcmul')) {
+            $this->markTestSkipped('Test will only work when bcmath-extension is enabled');
```

Comment:
use `@requires function bcmul` (or `@requires extension bcmath`) instead.

Reply:
should be removed now that the code doesn't use bcmath functions anymore